packages/design-system/src/components/Checkbox/Checkbox.tsx

```diff
@@ -96,6 +98,7 @@ const CheckboxRoot = styled(Checkbox.Root)`
     height: 100%;
     min-width: 44px;
     min-height: 44px;
+    pointer-events: none;
```

Comment:
We can't do that because it's not just a visual change about what the cursor looks like, it makes the area not clickable. And as indicated in the comment above we do want to increase the touch target to make it easier to click, especially on mobile.

So I would recommend changing the approach and removing `cursor: pointer` from the root, and instead only applying it to the icon. Then you wouldn't need to change this pseudo element

Reply:
@remidej , I tried applying `cursor: pointer` to the icon, but it doesn’t show in the corners of the checkbox. To resolve this, I added it to the Checkbox Indicator, which ensures that it now encompasses the entire area.

Please let me know if you have any additional thoughts or suggestions!